tests/e2e/Employee.test.ts

```diff
@@ -0,0 +1,102 @@
+import { test, Page, expect } from "@playwright/test";
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I think we can remove this line because page is being used

Reply:
Done in https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/238/commits/275ee76be87561638c15756f2f7840eb129261b1